src/gcodePlanner.cpp

```diff
@@ -238,7 +238,7 @@ void GCodePlanner::moveInsideCombBoundary(int distance)
     }
 }
 
-GCodePath& GCodePlanner::addTravel(Point p)
+GCodePath& GCodePlanner::addTravel(Point p, bool going_to_outer_wall)
```

Comment:
I would call the new variable `force_retraction` and retrieve the setting "combing_retract_before_outer_wall" higher up in the call hierarchy. Perhaps as high up as `addPolygonsByOptimizer`.